bin/etcd-manage

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -eo pipefail
+
+preStart() {
+  # pull ip address for current machine and set in environment
+  export ETCD_IP=$(hostname -i)
```

Comment:
The `port` configuration you have for etcd in the Compose file will allocate a public IP address for each of the etcd containers. Will `hostname -i` definitely get us the right IP address here? Come to think of it, should we be exposing the public IP at all?